config.py

```diff
@@ -1 +1,17 @@
 ROTATE_MAX_CHARS = 15
+
+SUBGROUPS = {'EE': ['DiddleDad', 'fgsrfug', 'Selfishshellfish', 'aaronschraner'],
```

Comment:
We may want to look into if it is possible to dynamically get these rather than hard-coding them into the config file. It may not be possible - or at least not easily.